src/components/Header/Header.css

```diff
@@ -1,6 +1,6 @@
 .header {
   color: rgb(88, 88, 88);
-  margin: 15px;
+  margin-top: 45px;
```

Comment:
Let's increase this to `60px` to add a bit more space between navbar and header.